src/main/java/filter/expression/Qualifier.java

```diff
@@ -91,21 +92,18 @@ public static FilterExpression replaceMilestoneAliases(IModel model, FilterExpre
             repoIds.add(model.getDefaultRepo().toLowerCase());
         }
 
-        List<TurboMilestone> allMilestones = model.getMilestones().stream()
-                .filter(ms -> ms.getDueDate().isPresent())
-                .filter(ms -> repoIds.contains(ms.getRepoId().toLowerCase()))
-                .sorted((a, b) -> getMilestoneDueDateComparator().compare(a, b))
-                .collect(Collectors.toList());
-
-        Optional<Integer> currentMilestoneIndex = getCurrentMilestoneIndex(allMilestones);
+        List<TurboMilestone> milestonesOfReposInPanel = TurboMilestone.filterMilestonesOfRepos(
+                                                                                    model.getMilestones(), repoIds);
+        List<TurboMilestone> milestonesWithinAliasRange = getAliasableMilestones(milestonesOfReposInPanel);
```

Comment:
I forgot to point this out. Change the variable name to `aliasableMilestones` for consistency purpose.